storage/json-api/pom.xml

```diff
@@ -15,6 +15,15 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>2.5.1</version>
+        <configuration>
+          <source>1.8</source>
```

Comment:
You can specify this w/ the shortcut maven.compiler.source & maven.compiler.target  -- it's one less thing we need to track.